templates/mainapp/layout.html

```diff
@@ -136,12 +137,23 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
         </div>
     </div>
     </div>
-
+	<script>
```

Comment:
add this script [here](https://github.com/Praful932/Kitabe/blob/61f4fbf6d7e2366b11a563d17603d6098b3f5807/templates/mainapp/layout.html#L168) we already have a script tag right, so all js should be inside it no need to add another block.

Reply:
Ok sure!